### PR TITLE
[DOCS] Update attributes and text references for Elastic Defend integration

### LINKED ISSUE #2464: [DOCS] Update attributes and text references for Elastic Defend integration
## Description 

- Parent docs meta: https://github.com/elastic/security-docs/issues/2463
- Related dev issue: https://github.com/elastic/security-team/issues/4759

Pending the new attribute created in https://github.com/elastic/docs/issues/2516, we need to update all existing docs that reference the "Endpoint and Cloud Security" integration (`{endpoint-cloud-sec}`) to reference the new name and attribute: "Elastic Defend" and `{elastic-defend}`. 

## Important notes

- Although we recently cut down on hard-written references in [the previous renaming in 8.3](https://github.com/elastic/security-docs/issues/2046), it's still worthwhile to run a search for both attributes AND text using the old names.

- Merge changes into `main` and backport to `8.5`. All docs prior to `8.5` will still reference the old attributes (`{endpoint-cloud-sec}` for 8.3 and 8.4, `{endpoint-sec}` for 8.2 and earlier).

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #2464.

The endpoint integration "Endpoint and Cloud Security" (previously "Endpoint Security") is being renamed to "Elastic Defend." This PR updates attributes and text references to use the new name.

This also includes a few screenshot updates on the [Endpoints page](https://security-docs_2476.docs-preview.app.elstc.co/guide/en/security/master/admin-page-ov.html), because it was easier to update everything in one PR. A few other screenshots remain to be updated on the integration installation page, but those will be done separately as part of revising the "guided onboarding" process for Elastic Defend (issue [#2457](https://github.com/elastic/security-docs/issues/2457))

### Previews

This PR updates a LOT of files. Refer to the [HTML diff](https://security-docs_2476.docs-preview.app.elstc.co/diff) and the[ Files changed tab](https://github.com/elastic/security-docs/pull/2476/files).

### Checklist

- [x] Confirmed no more references: `endpoint-sec`, `endpoint-cloud-sec`, `Endpoint and Cloud Security`
- [x] References to `Endpoint Security`
   - [x] Name of rule is still "Endpoint Security"
   - [x] General industry term
   - [x] Old SIEM guide files
- [x] Additional screenshots to update:
   - [x] Policies page | policy-list.png
   - [x] T'shooting | unhealthy-agent-fleet.png
   - [x] T'shooting | endpoints-transform-failed.png

### CODE REVIEW

**Review by kevinlog — APPROVED** [review]
LGTM!

**Review by benironside — APPROVED** [review]
LGTM!

File: docs/management/admin/policy-list.asciidoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [chapter]
 = Policies
 
-The **Policies** page lists all of the integration policies configured for {endpoint-cloud-sec}. 
+The **Policies** page lists all of the integration policies configured for {elastic-defend}. 
 
 Click on an integration policy's name to configure its settings. For more information on configuring an integration policy, refer to <<configure-endpoint-integration-policy>>.
```
Comment (nastasha-solomon) [inline docs/management/admin/policy-list.asciidoc:7]: Does the following screenshot need to be refreshed?

File: docs/getting-started/install-endpoint.asciidoc [hunk 2]
```diff
@@ -1,9 +1,9 @@
 [[install-endpoint]]
-= Configure and install the {endpoint-cloud-sec} integration
+= Configure and install the {elastic-defend} integration
```
Comment (nastasha-solomon) [inline docs/getting-started/install-endpoint.asciidoc:2]: IIRC, you said the screenshots in this topic will be updated at a later point once integration onboarding efforts concluded?

File: docs/troubleshooting/management/ts-management.asciidoc [hunk 3]
```diff
@@ -12,34 +12,34 @@ This topic covers common troubleshooting issues when using {elastic-sec} <<sec-m
 .Unhealthy {agent} status
 [%collapsible]
 ====
-In some cases, an `Unhealthy` {agent} status may be caused by a failure in the {endpoint-cloud-sec} integration policy. In this situation, the integration and any failing features are flagged on the agent details page in {fleet}. Expand each section and subsection to display individual responses from the agent.
+In some cases, an `Unhealthy` {agent} status may be caused by a failure in the {elastic-defend} integration policy. In this situation, the integration and any failing features are flagged on the agent details page in {fleet}. Expand each section and subsection to display individual responses from the agent.
 
 TIP: Integration policy response information is also available from the *Endpoints* page in the {security-app} (*Manage* -> *Endpoints*, then click the link in the *Policy status* column).
 
 [role="screenshot"]
```
Comment (nastasha-solomon) [inline docs/troubleshooting/management/ts-management.asciidoc:19]: This image still shows the old cloud integration in the agent policy. Feel free to ignore this comment if you're already tracking this :)

File: docs/troubleshooting/management/ts-management.asciidoc [hunk 4]
```diff
@@ -67,7 +67,7 @@ image::images/endpoints-transform-failed.png[Endpoints page with Required transf
 To restart a transform that’s not running:
```
Comment (nastasha-solomon) [inline docs/troubleshooting/management/ts-management.asciidoc:67]: The screenshot above these steps (endpoints-transform-failed.png) looks outdated. It's still showing `endpoint security` in the Endpoints page description.

**Review by nastasha-solomon — COMMENTED** [review]
Looks great and thank you for this massive effort, @joepeeples! I left a handful of comments about screenshots. Feel free to ignore any that are already on your radar.

File: docs/getting-started/install-endpoint.asciidoc (hunk 2 above)
Comment (joepeeples) [reply docs/getting-started/install-endpoint.asciidoc:2]: Correct - the screenshots on this page will be updated as part of https://github.com/elastic/security-docs/issues/2457.

File: docs/management/admin/policy-list.asciidoc (hunk 1 above)
Comment (joepeeples) [reply docs/management/admin/policy-list.asciidoc:7]: Yes, good catch! Prob needed to be updated back in 8.3, but we'll get it done now. 😉

Comment (joepeeples) [reply docs/management/admin/policy-list.asciidoc:7]: Screenshot updated!

File: docs/troubleshooting/management/ts-management.asciidoc (hunk 3 above)
Comment (joepeeples) [reply docs/troubleshooting/management/ts-management.asciidoc:19]: Screenshot updated!

File: docs/troubleshooting/management/ts-management.asciidoc (hunk 4 above)
Comment (joepeeples) [reply docs/troubleshooting/management/ts-management.asciidoc:67]: Screenshot updated!

**Review by nastasha-solomon — APPROVED** [review]
🚀🚀🚀

File: docs/getting-started/configure-integration-policy.asciidoc [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+After the {agent} is installed with the {elastic-defend} integration, several protections features — including
 preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled
 on protected hosts (some features require a Platinum or Enterprise license). If needed, you can update the
 integration policy to configure protection settings, event collection, antivirus settings, trusted applications,
 event filters, host isolation exceptions, and blocked applications to meet your organization's security needs.
 
-You can also create multiple integration policies to maintain unique configuration profiles. To create an additional {endpoint-cloud-sec} integration policy, go to **Management** -> **Integrations**, then follow the steps for <<add-security-integration, adding the {endpoint-cloud-sec} integration>>.
+You can also create multiple integration policies to maintain unique configuration profiles. To create an additional {elastic-defend} integration policy, go to **Management** -> **Integrations**, then follow the steps for <<add-security-integration, adding the {elastic-defend} integration>>.
```
Comment (jmikell821) [inline docs/getting-started/configure-integration-policy.asciidoc:10]: Not sure if we should refer to our policies as `{elastic-defend} integration policy`, `integration policy`, or if it matters much. But in the first sentence it is sans {`elastic-defend}`; in the second it is not.

File: docs/management/admin/event-filters.asciidoc [hunk 6]
```diff
@@ -82,7 +82,7 @@ To edit an event filter:
 [discrete]
 [[delete-event-filter]]
 === Delete an event filter
-You can delete an event filter, which removes it entirely from all {endpoint-cloud-sec} policies.
+You can delete an event filter, which removes it entirely from all {elastic-defend} policies.
```
Comment (jmikell821) [inline docs/management/admin/event-filters.asciidoc:85]: Just adding for consistency but feel free to omit. 

```suggestion
You can delete an event filter, which removes it entirely from all {elastic-defend} integration policies.
```

File: docs/management/admin/host-isolation-exceptions.asciidoc [hunk 7]
```diff
@@ -49,7 +49,7 @@ To edit a host isolation exception:
 [discrete]
 [[delete-host-isolation-exception]]
 === Delete a host isolation exception
-You can delete a host isolation exception, which removes it entirely from all {endpoint-cloud-sec} policies.
+You can delete a host isolation exception, which removes it entirely from all {elastic-defend} policies.
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-exceptions.asciidoc:52]:
```suggestion
You can delete a host isolation exception, which removes it entirely from all {elastic-defend} integration policies.
```

File: docs/management/admin/trusted-apps.asciidoc [hunk 8]
```diff
@@ -70,7 +70,7 @@ To edit a trusted application:
 [discrete]
 [[delete-trusted-app]]
 === Delete a trusted application
-You can delete a trusted application, which removes it entirely from all {endpoint-cloud-sec} policies.
+You can delete a trusted application, which removes it entirely from all {elastic-defend} policies.
```
Comment (jmikell821) [inline docs/management/admin/trusted-apps.asciidoc:73]:
```suggestion
You can delete a trusted application, which removes it entirely from all {elastic-defend} integration policies.
```

File: docs/troubleshooting/management/ts-management.asciidoc [hunk 9]
```diff
@@ -76,5 +76,5 @@ To restart a transform that’s not running:
 image::images/transforms-start.png[Transforms page with Start option selected]
 
 . On the confirmation message that displays, click *Start* to restart the transform.
-. The transform’s status changes to `started`. Refresh the page if you don't see the change.
+. The transform’s status changes to `started`. Refresh the page if you don't observe the change.
```
Comment (jmikell821) [inline docs/troubleshooting/management/ts-management.asciidoc:79]: Alt suggestion to `observe`:
```suggestion
. The transform’s status changes to `started`. If it doesn't change, refresh the page.
```

**Review by jmikell821 — COMMENTED** [review]
Looks great! Thanks for these changes, @joepeeples. 🌟 Just left a few comments.

File: docs/getting-started/configure-integration-policy.asciidoc (hunk 5 above)
Comment (joepeeples) [reply docs/getting-started/configure-integration-policy.asciidoc:10]: Both sentences would be good with `{elastic-defend} integration policies/policy`, for clarity and consistency; will make that change!

Comment (joepeeples) [reply docs/getting-started/configure-integration-policy.asciidoc:10]:
```suggestion
You can also create multiple {elastic-defend} integration policies to maintain unique configuration profiles. To create an additional {elastic-defend} integration policy, go to **Management** -> **Integrations**, then follow the steps for <<add-security-integration, adding the {elastic-defend} integration>>.
```

**Pull request merged**